### DISCUSSION TITLE
NexusPrismaTypes plural duplicates

### DISCUSSION THREAD

**Discussion by TomasMoskala** [prisma/prisma#3196] [Q&A] [answered]
Hi,
is it not allowed to have tables ending with "s"? I get errors with duplicates.

MySQL
```
CREATE TABLE `activity_gps` (
  `activity_gps_id` BIGINT NOT NULL AUTO_INCREMENT,
  `id_activity` INT NOT NULL,
  `activity_gps_longitude` DECIMAL(7,4) NOT NULL,
  `activity_gps_latitude` DECIMAL(6,4) NOT NULL,
  `activity_gps_altitude` SMALLINT NULL,
PRIMARY KEY (`activity_gps_id`)) ENGINE=InnoDB DEFAULT CHARSET=utf8;

```
schema.prisma
```
model activity_gps {
  activity_gps_altitude  Int?
  activity_gps_id        Int      @default(autoincrement()) @id
  activity_gps_latitude  Float
  activity_gps_longitude Float
  id_activity            Int
  activity               activity @relation(fields: [id_activity], references: [activity_id])

  @@index([id_activity], name: "fk_activity_activity_gps")
}
```

nexus-prisma-typegen/index.d.ts
```

interface NexusPrismaTypes {
  Query: {
    activityGps: 'activity_gps'
    activityGps: 'activity_gps'
    activityPhoto: 'activity_photo'
    activityPhotos: 'activity_photo'

```
- apparently a problem with the plural generation

Build errors 
node_modules/@types/nexus-prisma-typegen/index.d.ts:760:5 - error TS2300: Duplicate identifier 'activityGps'.
760     activityGps: 'activity_gps'
        ~~~~~~~~~~~

node_modules/@types/nexus-prisma-typegen/index.d.ts:761:5 - error TS2300: Duplicate identifier 'activityGps'.
761     activityGps: 'activity_gps'
        ~~~~~~~~~~~

**Comment by ryands17** [comment, marked as answer]
Hwy @TomasMoskala 👋 
This is an issue with Nexus and the underlying issue is [here](https://github.com/graphql-nexus/nexus-plugin-prisma/issues/537) that you can check out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
